specs/confluent-kafka-rest/SOURCES/kafka-rest.init

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# kafka-rest        Startup script for kafka-rest
+
+# chkconfig: - 85 15
+# processname: kafka-rest
+# config: /etc/kafka-rest/kafka-rest.conf
+# config: /etc/sysconfig/kafka-rest
+# pidfile: /var/run/kafka-rest.pid
+# description: Confluent REST Proxy for Kafka
+
+### BEGIN INIT INFO
+# Provides: kafka-rest
+# Required-Start: $local_fs $remote_fs $network
+# Required-Stop: $local_fs $remote_fs $network
+# Default-Start: 2 3 4 5
+# Default-Stop: 0 1 6
+# Short-Description: start and stop kafka-rest
+### END INIT INFO
+
+###############################################################################
+
+source /etc/init.d/kaosv
+
+###############################################################################
+
+kafka_rest_user="kafka-rest"
+binary=${BINARY:-/usr/bin/kafka-rest-start}
+conf_file=${CONF_FILE:-/etc/kafka-rest/kafka-rest.conf}
+
+###############################################################################
+
+kv[user]="$kafka_rest_user"
+kv[prog_name]="kafka-rest"
+kv[search_pattern]="$conf_file"
```

Comment:
`search_pattern` is deprecated and not required in the latest version of kaosv. You can just define version ≥ 2.15 in `Requires` section and drop this definition.

Reply:
Fixed (8d618a1).